Structure_Engine/Compute/ReinforcementDensity.cs

```diff
@@ -115,81 +115,15 @@ public static ReinforcementDensity ReinforcementDensity(BarRequiredArea barRequi
 
         /***************************************************/
 
+        [NotImplemented]
         [Description("Calculates the ReinforcementDensity of a MeshRequiredArea from a given Bar and Material.")]
         [Input("meshRequiredArea", "The BarRequiredArea containing the reinforced areas and ids for the Bar and Material.")]
         [Input("elements", "The elements to search for the relevant AreaElement associated with the MeshRequiredArea.")]
         [Input("materials", "The Materials to search for the relevant Material associated with the MeshRequiredArea.")]
         [Output("reinforcementDensity", "The ReinforcementDensity calculated using the inputs provided.")]
         public static ReinforcementDensity ReinforcementDensity(MeshRequiredArea meshRequiredArea, List<IAreaElement> elements, List<IMaterialFragment> materials)
         {
-            Reflection.Compute.RecordError("Requires an implementation of Regions before the required area can be extrapolated.");
             return null;
```

Comment:
I believe there should be some sort of error here to inform the consumer why `null` is returned

Reply:
It has a `[Not Implemented]` attribute at the top so should not even appear in the UI - it was more a placeholder so we could develop the method later. I can remove it if you'd like.

Reply:
I guess one could still try to call this method from the code. Of cource not too likely but better be on the safe side IMO. Anyways, it is a minor one, I would not bother about it now, rather add it to the batch in case other issues pop up during the review.